CODEOWNERS

```diff
@@ -1,4 +1,33 @@
-/core/ @se3000 @RyanRHall @spooktheducks @samsondav @j16r @connorwstein @PiotrTrzpil @archseer
+# Services
+/core/services/bulletprooftxmanager @PiotrTrzpil @samsondav
+/core/services/cron @spooktheducks @samsondav
+/core/services/directrequest @PiotrTrzpil @spooktheducks @j16r @connorwstein @samsondav
+/core/services/eth @PiotrTrzpil @spooktheducks @samsondav
+/core/services/fluxmonitorv2 @jkongie @PiotrTrzpil @spooktheducks @connorwstein
+/core/services/gasupdater @samsondav @PiotrTrzpil
+/core/services/headtracker @PiotrTrzpil @samsondav
+/core/services/health @archseer @samsondav
+/core/services/job @spooktheducks @connorwstein @archseer
+/core/services/keeper @RyanRHall @samsondav
+/core/services/log @PiotrTrzpil @spooktheducks
+/core/services/offchainreporting @spooktheducks @connorwstein @samsondav
+/core/services/periodicbackup @PiotrTrzpil @samsondav
+/core/services/pipeline @spooktheducks @connorwstein @archseer
+/core/services/postgres
```

Comment:
is this the same behaviour as not mentioning the package here? And would it imply that everyone is an owner of it or no one is?

Reply:
I would say that this implies there are no owners. 

 I've left it in for us to fill out as we need to but no one has claimed those as of yet.